CHANGELOG

```diff
@@ -1,34 +1,41 @@
+  apps into separate insert/replace and remove operations. This makes the
+  overall update not atomic, but as removes are unlikely here, we can now get
+  away with a simple multi-document insert-replace operation instead of a
+  truncate and an exclusive transaction, which was used before.
 
+* Added metrics for document read and write operations:
+  
   - `arangodb_document_writes: Total number of document write operations
     (successful and failed) not performed by synchronous replication.
   - `arangodb_document_writes_replication`: Total number of document write
-    operations (successful and failed) by synchronous replication.
+    operations (successful and failed) by cluster synchronous replication.
   - `arangodb_collection_truncates`: Total number of collection truncate
-    operations (successful and failed) not performed by synchronous replication.
+    operations (successful and failed) not performed by cluster synchronous
+    replication.
   - `arangodb_collection_truncates_replication`: Total number of collection
-    truncate
-    operations (successful and failed) by synchronous replication.
-  - `arangodb_document_read_time`: Execution time histogram of all document read
-    operations (successful and failed) [s].
-  - `arangodb_document_insert_time`: Execution time histogram of all document
-    insert
-    operations (successful and failed) [s].
-  - `arangodb_document_replace_time`: Execution time histogram of all document
-    replace
-    operations (successful and failed) [s].
-  - `arangodb_document_remove_time`: Execution time histogram of all document
-    remove
-    operations (successful and failed) [s].
-  - `arangodb_document_update_time`: Execution time histogram of all document
-    update
-    operations (successful and failed) [s].
+    truncate operations (successful and failed) by synchronous replication.
+  - `arangodb_document_read_time`: Execution time histogram of all document
+    primary key read operations (successful and failed) [s]. Note: this
+    does not include secondary index lookups, range scans and full collection
+    scans.
+  - `arangodb_document_insert_time`: Execution time histogram of all
+    document insert operations (successful and failed) [s].
+  - `arangodb_document_replace_time`: Execution time histogram of all
+    document replace operations (successful and failed) [s].
+  - `arangodb_document_remove_time`: Execution time histogram of all
+    document remove operations (successful and failed) [s].
+  - `arangodb_document_update_time`: Execution time histogram of all
+    document update operations (successful and failed) [s].
   - `arangodb_collection_truncate_time`: Execution time histogram of all
     collection truncate operations (successful and failed) [s].
 
+  The timer metrics are turned off by default, and can be enabled by setting
+  the startup option `--server.export-read-write-metrics true`.
+
```

Comment:
This piece looks a bit unrelated.
Is it possible that this is a hidden merge conflict?

Reply:
The CHANGELOG change is undeed unrelated to this PR, but I put it in as a clarification.
The clarification was added to devel already, but was missing in 3.7.
So it is intentional as it is.
Of course I could have split this into 2 separate PRs, just didn't due to lack of :hourglass:

Reply:
ok 👍